sickrage/show/recommendations/trakt.py

```diff
@@ -90,33 +90,34 @@ def fetch_popular_shows(self, page_url=None, trakt_list=None):  # pylint: disabl
         :throw: ``Exception`` if an Exception is thrown not handled by the libtrats exceptions
         """
         trending_shows = []
+        blacklist = ''
```

Comment:
@duramato @fernandog this could be a possible fix. But then I still don't know why it isn't initializing the blacklist var.